### PR TITLE
UnknownFObject

### PR DESCRIPTION
**Pull request by xuerongCode**
add UnknownFObject model
add OutputJSON interface

### CODE REVIEW

File: src/foam/lib/json/OutputJSON.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+foam.INTERFACE({
```
Comment (kgrgreer) [inline src/foam/lib/json/OutputJSON.js:6]: Insert blank line.
Did you remove src/foam/lib/json/OutputJSON.java?

Comment (xuerongCode) [reply src/foam/lib/json/OutputJSON.js:6]: no

File: src/foam/lib/json/OutputJSON.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  package: 'foam.lib.json',
```
Comment (kgrgreer) [inline src/foam/lib/json/OutputJSON.js:7]: Why did you need to convert to a FOAM interface?

Comment (xuerongCode) [reply src/foam/lib/json/OutputJSON.js:7]: I get compile error if I do not have foam.lib.json.OutputJSON interface model.

Comment (xuerongCode) [reply src/foam/lib/json/OutputJSON.js:7]: AssertionError [ERR_ASSERTION]: Could not find any registered class for foam.lib.json.OutputJSON

File: src/foam/lib/json/OutputJSON.js (hunk 1 above)
Comment (kgrgreer) [reply src/foam/lib/json/OutputJSON.js:6]: You should.

Comment (xuerongCode) [reply src/foam/lib/json/OutputJSON.js:6]: already done
